services/metadata/frontend/Footer.svelte

```diff
@@ -0,0 +1,19 @@
+<div class="footer">Footer...</div>
+
+<style>
+  .footer {
+    background-color: rgb(185, 184, 184);
+    /* background-color: rgb(89, 73, 167); */ /* DaSCH Purple */
+    color: whitesmoke;
+    font-weight: bold;
+    /* display: inline-flex; */
+    width: 100%;
+    position: fixed;
+    bottom: 0px;
+    padding: 16px;
+    color: rgb(90, 90, 90);
+    /* justify-content: space-between; */
+    /* align-items: center; */
+    /* vertical-align: bottom; */
+  }
+</style>
```

Comment:
It's good practice to leave empty line on the bottom of files. I've already added the rule to automate that, but is still in my branch ;)

Reply:
darn... normally I'm the first one to hate people who don't.